### PR TITLE
Check virtool.ca for reference releases

### PR DESCRIPTION
**Pull request by SnickerDoodlePop**

### CODE REVIEW

File: tests/references/test_db.py [hunk 1]
```diff
@@ -177,3 +192,54 @@ async def test_delete_member(field, snapshot, mongo):
 
     assert subdocument_id == snapshot
     assert await mongo.references.find_one() == snapshot
+
+
+@pytest.mark.parametrize("status", [200, 304, 404])
+async def test_fetch_and_update_release(mongo, status, fake_app, snapshot, static_time):
+    await startup_http_client(fake_app)
+
+    match status:
```
Comment (igboyes) [inline tests/references/test_db.py:201]: This can be simplified to a few `if` statements.

File: virtool/references/db.py [hunk 2]
```diff
@@ -399,30 +482,41 @@ async def fetch_and_update_release(
...
+
+        updated = await get_releases_from_virtool(
             client, document["remotes_from"]["slug"], etag
         )
 
+        # ditto
+
         if updated:
-            updated = virtool.github.format_release(updated)
+            updated = format_latest_release(updated)
```
Comment (igboyes) [inline virtool/references/db.py:504]: Do you see a problem with variable naming here?

**Review by igboyes — CHANGES_REQUESTED** [review]
Black formatting does not seem to be working for you. Please resolve.

File: virtool/references/db.py [hunk 3]
```diff
@@ -399,30 +482,41 @@ async def fetch_and_update_release(
     )
 
     release = document.get("release")
-    etag = virtool.github.get_etag(release)
+
+    # change below here
```
[the remark this answers is not in the document]
Comment (SnickerDoodlePop) [reply virtool/references/db.py:486]: Will be removed before merging. Apologies they are still included; these comments are only for myself while working.

**Review by igboyes — COMMENTED** [review]
Have you run this in Minikube to make sure it works when deployed?

**Review by igboyes — APPROVED** [review]

**Pull request merged**